README.md

```diff
@@ -58,7 +58,7 @@ Install the required dependencies using [Homebrew](https://brew.sh/):
 
   ```bash
   brew install cmake graphviz libpng ninja protobuf wget
-  brew install --with-toolchain llvm@6
+  brew install --with-toolchain llvm
```

Comment:
please do not remove this, Glow cannot run on llvm7 which is current when `brew install`.

Reply:
My mistake, I will update the PR shortly.